Database/Updates/World/2026-08-07-03-GrumpyPack-Quests.sql

```
@@ -0,0 +1,69 @@
+-- Carry-over: 65 quest rows.
+DELETE FROM `quest` WHERE `name` IN ('BiakaM10KillCount', 'BiakaM10KillTask', 'GigasRaiderM11KillCount', 'GigasRaiderM11KillTask', 'GreenPhyntosWaspM9KillCount', 'GreenPhyntosWaspM9KillTask', 'HeaItealuanM9KillCount', 'HeaItealuanM9KillTask', 'IslandArmoredilloM10KillCount', 'IslandArmoredilloM10KillTask', 'MagmaGolemM10KillCount', 'MagmaGolemM10KillTask', 'OlthoiLancerM9KillCount', 'OlthoiLancerM9KillTask', 'PutridMoarsmanM10KillCount', 'PutridMoarsmanM10KillTask', 'SiessaSclavusSoldierM11KillCount', 'SiessaSclavusSoldierM11KillTask', 'SpectralBloodmageM11KillCount', 'SpectralBloodmageM11KillTask', 'UberPenguinM11KillCount', 'UberPenguinM11KillTask', 'WaveGolemM9KillCount', 'WaveGolemM9KillTask', 'ZC_TouTou_A1Completed', 'ZC_TouTou_A1Gate', 'ZC_TouTou_A1Lantern', 'ZC_TouTou_A1Stone', 'ZC_TouTou_A2Completed', 'ZC_TouTou_A3Completed', 'ZC_TouTou_A3L1', 'ZC_TouTou_A3L2', 'ZC_TouTou_A3L3', 'ZC_TouTou_A3L4', 'ZC_TouTou_A3L5', 'ZC_TouTou_A3Lanterns', 'ZC_TouTou_A4Completed', 'ZC_TouTou_A4R1', 'ZC_TouTou_A4R2', 'ZC_TouTou_A5Completed', 'ZC_TouTou_A5Engineers', 'ZC_TouTou_C1Completed', 'ZC_TouTou_C2Completed', 'ZC_TouTou_C3Completed', 'ZC_TouTou_C4Completed', 'ZC_TouTou_C5Completed', 'ZC_TouTou_C6Completed', 'ZC_TouTou_D1Completed', 'ZC_TouTou_KillBones', 'ZC_TouTou_KillBonesCompleted', 'ZC_TouTou_KillChampions', 'ZC_TouTou_KillChampionsCompleted', 'ZC_TouTou_KillDrowned', 'ZC_TouTou_KillDrownedCompleted', 'ZC_TouTou_KillGolems', 'ZC_TouTou_KillGolemsCompleted', 'ZC_TouTou_KillMosswarts', 'ZC_TouTou_KillMosswartsCompleted', 'ZC_TouTou_KillReef', 'ZC_TouTou_KillReefCompleted', 'ZC_TouTou_KillShadows', 'ZC_TouTou_KillShadowsCompleted', 'ZC_TouTou_KillSwarmCompleted', 'ZC_TouTou_KillWaspQueen', 'ZC_TouTou_KillWasps');
```

SQLFluff (4.3.0)

[error] The 'WHERE' keyword should always start a new line.

(LT14)